app/paths.js

```diff
@@ -141,5 +141,9 @@ module.exports = {
     editAmount: '/create-payment-link/manage/edit/amount/:productExternalId'
   },
   feedback: '/feedback',
+  partnerApp: {
+    linkAccount: '/link-account',
```

Comment:
We currently don't test navigating directly to URLs without valid session (as it is partly handled in other tests), but if you add tests for this case then it will be awesome!
Not sure how much time it will take as well, so something to discuss and clarify if we need it.

Reply:
I believe some of this functionality is tested when we set a valid request with account, session etc. but we can sit and see if we can find a way to test all the paths.

Reply:
I changed the tests for middleware to include the routing bit as well and now there are clear tests asserting on user being redirected to login if not authenticated.